File: resolucoes/JavaScript/lista01/ex005.js
const prompt = require('prompt-sync')();

function main() {
    const nota1 = Number(prompt("Qual foi sua primeira nota?"))
    const nota2 = Number(prompt("Qual foi sua segunda nota?"))
    const media = (nota1 + nota2) / 2
    prompt(`A soma entre ${nota1} e ${nota2} é ${media}`)
}

module.exports = {
    main
}

File: resolucoes/Lua/lista01/ex005.lua
print("Digite a primeira nota: ")
nota1 = tonumber(io.read()) -- Lê a primeira nota digitada pelo usuário e converte para um número

print("Digite a segunda nota: ")
nota2 = tonumber(io.read()) -- Lê a segunda nota digitada pelo usuário e converte para um número

media = (nota1 + nota2) / 2 -- Calcula a média das duas notas

print("A media entre " .. nota1 .. " e " .. nota2 .. " e igual a " .. media) -- Mostra a média na tela